### ISSUE TITLE
handle multiple header values

### ISSUE THREAD

**Issue by yanns** [LukeMathWalker/wiremock-rs#111]
When I try to assert on a header that can be appears multiple times in a request, it seems that wiremock only consider the first instance.
When I send:
```
&service_name_headers = {
    "x-service-name": "TJ2K4cgLPB",
    "x-service-name": "kAQmNfsbYjWVdji",
    "x-service-name": "IGqauf",
    "x-service-name": "YNdLE3fEMi0",
}
```
The error from wiremock:
```
failed:
- Mock #0.
	Expected range of matching incoming requests: == 1
	Number of matched incoming requests: 0

Received requests:
- Request #1
	GET http://localhost/temporibus/feature-tags
accept: application/json
host: 127.0.0.1:60250
x-service-name: TJ2K4cgLPB
```
`x-service-name` appears only once here.
When I check on "real" server, I can observe that all header values are present.

**Comment by LukeMathWalker** [comment]
How are you passing the multiple headers in the request?

**Comment by LukeMathWalker** [comment]
It should be fixed in f8eb9a5 - I've released it as part of 0.5.18 👍🏻

**Comment by yanns** [comment]
I'm passing the multi-value with `HeaderMap`.
I can confirm that the fix is working. Thanks a lot!
